### PR TITLE
Add post model and unittest

### PR DESCRIPTION
**Pull request by sh92**

### CODE REVIEW

File: hyde/models/jekyll.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from ..extensions import db
+from .post import Post
+
+
+class Jekyll(Post):
+    __tablename__ = 'Jekyll'
```
Comment (corona10) [inline hyde/models/jekyll.py]: __tablename__ = 'JekyllPost' might be better 👍

File: hyde/models/jekyll.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from ..extensions import db
+from .post import Post
+
+
+class Jekyll(Post):
```
Comment (corona10) [inline hyde/models/jekyll.py]: Change class name into JekyllPost.

File: hyde/models/jekyll.py (hunk 1 above)
Comment (sh92) [inline hyde/models/jekyll.py]: Gotcha!

File: hyde/models/jekyll.py (hunk 2 above)
Comment (sh92) [inline hyde/models/jekyll.py]: LGTM

**Comment by corona10** [comment]
Rejected.

We need to see a code coverage integration on PR system.
Can you take a look this issue.

I want to see a code coverage for every single PR.

**Comment by corona10** [comment]
@sh92 Can you merge your branch with master. And then let's proceed PR.

**Comment by corona10** [comment]
LGTM

**Pull request merged**